sample_files/site_theme/override.css

```diff
@@ -8,8 +8,8 @@ body {
     background: rgba(240,240,240, 0.85) !important;
 }
 
-/* Force sidebars to 220px to line up with the image in top_bar.html */
```

Comment:
I think we should keep the comments here in CSS. The CSS files I've seen on Submitty have comments for what does what.